google/resource_compute_instance_test.go

```diff
@@ -1187,6 +1248,49 @@ func testAccComputeInstance_attachedDisk(disk, instance string) string {
 	}`, disk, instance)
 }
 
+func testAccComputeInstance_bootDisk(instance string) string {
+	return fmt.Sprintf(`
+	resource "google_compute_instance" "foobar" {
+		name         = "%s"
+		machine_type = "n1-standard-1"
+		zone         = "us-central1-a"
+
+		boot_disk {
+			initialize_params {
+				image = "debian-8-jessie-v20160803"
+			}
+			disk_encryption_key_raw = "SGVsbG8gZnJvbSBHb29nbGUgQ2xvdWQgUGxhdGZvcm0="
+		}
+
+		network_interface {
+			network = "default"
+		}
+	}`, instance)
```

Comment:
Nit: can the ` be moved to the next line? This would also make using these tests in .tf configs easier.